cylc/flow/config.py

```diff
@@ -1786,10 +1786,10 @@ def _check_task_event_handlers(self, taskdef: 'TaskDef') -> None:
                     try:
                         handler_template % handler_data
                     except (KeyError, ValueError) as exc:
-                        raise WorkflowConfigError(
+                        LOG.warning(
                             f'bad task event handler template'
                             f' {taskdef.name}: {handler_template}: {repr(exc)}'
-                        ) from None
```

Comment:
We don't want to demote this error to a warning.

This error is for invalid handler templates, this check has been in place for a while I think. An invalid handler template will lead to a runtime error.

Reply:
This results in a logged ERROR at runtime but it doesn't otherwise affect the workflow.

The issue #6916 is arguably a bit ambiguous - it says "invalid event error" but not specifically "invalid event **names**".  Seeing this change, I assumed you must have had a few workflows with bad templates as well, that were not causing real problems.

But if that's not the case, then yes - best not to demote this one!

Reply:
> I assumed you must have had a few workflows with bad templates as well, that were not causing real problems.

We don't.